vendor/engines/split_accounts/app/services/split_accounts/journal_row_converter.rb

```diff
@@ -0,0 +1,43 @@
+module SplitAccounts
+
+  # TODO: use an instance rather than a class service object and store journal
+  # and possibly order_detail
+  class JournalRowConverter < ::JournalRowConverter
```

Comment:
I was thinking nucore-open would use (sudo code):
default: JournalRowConverter,
split_accounts: SplitAccounts::JournalRowConverter < JournalRowConverter

And nucore would use:
default: Nu::JournalRowConverter,
split_accounts:  Nu::SplitAccounts::JournalRowConvert < Nu::JournalRowConverter